app/models/concerns/episode_media.rb

```diff
@@ -3,19 +3,32 @@
 module EpisodeMedia
   extend ActiveSupport::Concern
 
+  def cut_media_version!
+    latest_version = media_versions.first
+    latest_media = latest_version&.media_resources || []
+    latest_ids = latest_media.map(&:id)
+
+    # backfill media_versions for newly completed media
+    if media_ready? && latest_ids != media_ids
+      new_version = media_versions.build
+      media.each { |m| new_version.media_version_resources.build(media_resource: m) }
+      new_version.save!
+      new_version
+    else
+      latest_version
+    end
+  end
+
+  def media_version_id
+    cut_media_version!&.id
```

Comment:
Unused yet - but I figured this would be the place to track which media version we've sent to Apple.  Add a column to one of the apple tables, to track what the version was the last time we ran.

Reply:
Do we need this if we're cutting a new version?

1) If the set of media files have new version, we mark the episode as ready for re-upload
2) A new version means we call `Episode#publish!`
3) The the delegated delivery step in publishing picks up the new enclosure url and re-uploads.